pkg/api/api.go

```diff
@@ -12,12 +12,14 @@ import (
 	"github.com/grafana/grafana/pkg/infra/log"
 	"github.com/grafana/grafana/pkg/middleware"
 	"github.com/grafana/grafana/pkg/models"
+	acmiddleware "github.com/grafana/grafana/pkg/services/accesscontrol/middleware"
```

Comment:
We should remove all changes to `api.go` before merging, that's not ready to be merged.

Reply:
Sure, forget about that doing the refactor.